mptt/managers.py

```diff
@@ -53,51 +52,40 @@
     (
         SELECT m2.%(mptt_pk)s
         FROM %(mptt_table)s m2
-        WHERE m2.%(tree_id)s = %(mptt_table)s.%(tree_id)s
-          AND m2.%(left)s BETWEEN %(mptt_table)s.%(left)s
-                              AND %(mptt_table)s.%(right)s
+        WHERE m2.tree_id = %(mptt_table)s.tree_id
+          AND m2.lft BETWEEN %(mptt_table)s.lft
+                              AND %(mptt_table)s.rght
     )
 )"""
 
 
-def delegate_manager(method):
-    """
-    Delegate method calls to base manager, if exists.
-    """
-    @functools.wraps(method)
-    def wrapped(self, *args, **kwargs):
-        if self._base_manager:
-            return getattr(self._base_manager, method.__name__)(*args, **kwargs)
-        return method(self, *args, **kwargs)
-    return wrapped
-
-
 class TreeManager(models.Manager.from_queryset(TreeQuerySet)):
 
     """
     A manager for working with trees of objects.
     """
 
-    def contribute_to_class(self, model, name):
-        super(TreeManager, self).contribute_to_class(model, name)
+    @contextlib.contextmanager
+    def disable_mptt_updates(self):
+        # Preserve API
+        yield
```

Comment:
Maybe raise a deprecation warning here? It seems senseless to keep them around forever if they are defunct.